docs/content/latest/explore/query-1-performance/query-tuning-intro.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Tuning queries in YugabyteDB
+linkTitle: Introduction
+description: Tuning queries in YugabyteDB
+aliases:
+headerTitle: Tuning queries in YugabyteDB
+image: /images/section_icons/index/develop.png
+menu:
+  latest:
+    identifier: query-tuning-intro
+    parent: query-tuning
+    weight: 100
+isTocNested: true
+showAsideToc: true
+---
+
+A YugabyteDB cluster consists of multiple nodes running the YB-TServer server process. Due to its distributed nature, tuning a YugabyteDB database requires keeping in mind that, unlike most common databases where only a local machine influences a database query, data is stored across all the nodes of the cluster. This section provides an introduction to tuning YugabyteDB and the tools available.
+
+## Before you begin
+
+Before trying to optimize individual statements, first make sure the YugabyteDB cluster is running optimally:
+
+* Verify that all nodes that are supposed to be running are indeed running.
+* Verify that the YB-TServer and, where applicable, YB-Master processes are running on the nodes.
+
+To view the nodes and servers that make up your cluster, use the `yb-admin` command to request the master and tablet servers. For example:
+
+```sh
+$ ./bin/yb-admin -init_master_addrs=$(hostname):7100 list_all_masters
+```
+
+```output
+Master UUID                        RPC Host/Port           State       Role
+a637b88dfc0c4476862ca79872d763d7   172.158.22.229:7100     ALIVE       LEADER
+968eb4a7a2e24c3ab95929d3a888ed05   172.158.39.23:7100      ALIVE       FOLLOWER
+44e13e096955412a88c6ffd65a1e241c   172.158.50.212:7100     ALIVE       FOLLOWER
+```
+
+```sh
+$ ./bin/yb-admin -init_master_addrs=$(hostname):7100 list_all_tablet_servers
+```
+
+```output
+Tablet Server UUID               RPC Host/Port       Heartbeat delay Status   Reads/s  
+d80150e0eeda4477a231968440dd89a9 172.158.50.212:9100 0.20s           ALIVE    .. 
+721661fd5d2044179e03707a862aa578 172.158.22.229:9100 0.39s           ALIVE    .. 
+54e17c914c0f4679aa7b07a9c7a8ddf5 172.158.39.23:9100  0.48s           ALIVE    ..
+```
+
+Next, make sure enough disk space is available, elementary components such as CPU, disk, and the network do not report errors, and the operating system does not report any malfunction.
+
+For more information on troubleshooting cluster- and node-level issues, refer to [Troubleshoot YugabyteDB](../../../troubleshoot).
+
+Once you know the entire cluster is running correctly, you can move to statement tuning.
+
+## Tuning in YSQL / Postgres
+
+Each node running the YB-TServer process also runs an instance of Postgres (running as the YugabyteDB YSQL API). (This is often referred to as 'stateless Postgres', because each instance of Postgres runs independently.)
+
+To investigate a specific apparently slow running query, you first need to determine which node the query is running on. 
```

Comment:
"To identify the node on which the query is running, you can..."

Reply:
Waiting to hear back from @fritshoogland-yugabyte  or @epratt-yb on how to identify problem nodes in Core